cluster/examples/kubernetes/ceph/common.yaml

```diff
@@ -302,16 +302,8 @@ spec:
                           format: int32
                 security:
                   type: object
-                  properties:
-                    kms:
-                      type: object
-                      properties:
-                        connectionDetails:
-                          type: object
-                          nullable: true
-                          x-kubernetes-preserve-unknown-fields: true
-                        tokenSecretName:
-                          type: string
+                  nullable: true
```

Comment:
Why do we need to change this schema in common.yaml?

Reply:
actually, while testing test in 1.16 or every version it was failing due error ```testutil: cannot install helm chart with name : local/rook-ceph, version: v1.4.0-alpha.0.541.g35d0f0f, namespace: helm-ns  - . Error: release rook-ceph failed: CustomResourceDefinition.apiextensions.k8s.io "cephclusters.ceph.rook.io" is invalid: ```

Reply:
I'd suggest we leave this unchanged so that we still get the schema validation for the properties that are defined. The unknown fields in this case are just where we need to allow settings for different providers.

Reply:
Done.